week-3/Homework/mandatory/7-password-validator.js

```diff
@@ -23,47 +23,54 @@ PasswordValidationResult=  [false, false, false, false, true]
 */
 
 function validatePasswords(passwords) {
+  //var myS = ["Kbn.i", "Nvsk?", "csdR!"]
+  var strongRegex = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])\w{5,}$/;
 
+  // var access = strongRegex.test(myS);
+  var access = passwords.map(x => strongRegex.test(x));
```

Comment:
You can actually just return here like this
`return passwords.map(x => strongRegex.test(x));`

Give it a try!